packages/realm-react/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+        "plugin:@typescript-eslint/recommended",
+        "plugin:react-hooks/recommended"
+    ],
+    "rules": {
+        "@typescript-eslint/explicit-function-return-type": "off",
+        "react-hooks/rules-of-hooks": "error",
+        "react-hooks/exhaustive-deps": "warn"
```

Comment:
Did you want / need to add additional hooks there?

See the `additionalHooks` option mentioned at https://www.npmjs.com/package/eslint-plugin-react-hooks#user-content-advanced-configuration

Reply:
Not sure if the linter can pick that up.  These hooks are generated...

Reply:
I actually was able to get that to go.  There's a warning, but unsure how to address it

Reply:
Ok, so we would need this if our hooks took a function and array of dependencies as arguments, much like useEffect, useMemo, or useCallback.  Then it would be able to flag if the dependencies are being accounted for.  In our case, we can leave this out.